docs/source/AdministratorGuide/CommandReference/index.rst

```diff
@@ -59,7 +59,6 @@ Managing Resources:
     dirac-admin-ce-info
     dirac-admin-get-banned-sites
     dirac-admin-get-site-mask
-    dirac-admin-get-site-protocols
     dirac-admin-set-site-protocols
```

Comment:
This line also (it was twice in the list)

Reply:
The is "set" not get

Reply:
Ahhh, right, sorry